src/components/GoodCard.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+type Props = {
+  good: string;
+};
+
+export const GoodCard: React.FC<Props> = ({ good }) => (
+  <li data-cy="Good">{good}</li>
```

Comment:
According to the checklist, when rendering a list, each item should have a unique `key` prop. Since `GoodCard` is likely to be used within a list, it's important to accept a `key` prop and pass it to the `<li>` element.